### PR TITLE
[Gunk] Transaction: Update CreateTransactionRequest struct to include TransactionID

### PR DESCRIPTION
**Pull request by bobby-pratama**
Update transaction.gunk, add TransactionID property to  CreateTransactionRequest struct

[CEBCORP-63]

### CODE REVIEW

File: v1/transaction/transaction.gunk [hunk 1]
```diff
@@ -240,6 +240,9 @@ type CreateTransactionRequest struct {
 
 	// Remarks is an informational note about the transaction.
 	Remarks string `pb:"4" json:"remarks"`
+
+	// TransactionID is the unique identifier of a transaction.
+	TransactionID string `pb:"7" json:"transaction_id"`
```
Comment (Kunde21) [inline v1/transaction/transaction.gunk:245]: Let's add `SettlementRail` here to allow for multi-rail environments.

Comment (bobby-pratama) [reply v1/transaction/transaction.gunk:245]: @Kunde21 What is the data type for **SettlementRail** that we need to add?

Comment (Kunde21) [reply v1/transaction/transaction.gunk:245]: String type, because the entries will be different in each market.

Comment (bobby-pratama) [reply v1/transaction/transaction.gunk:245]: I already add the **SettlementRail** @Kunde21

Comment (bobby-pratama) [reply v1/transaction/transaction.gunk:245]: Btw, when we use the **SettlementRail** @Kunde21 ?

Comment (Kunde21) [reply v1/transaction/transaction.gunk:245]: There are multiple settlement rails in many markets, main two are realtime vs daily settlement.  They'll have different names in each market.

**Review by Kunde21 — APPROVED** [review]

**Pull request merged**